mcs/class/System.IO.Compression/SharpCompress/Archive/Zip/ZipWritableArchiveEntry.cs

```diff
@@ -46,11 +45,6 @@ public override long Size
             get { return size; }
         }
 
-        public override DateTime? LastModifiedTime
-        {
-            get { return lastModified; }
-        }
```

Comment:
This property wasn't being properly threaded, it already has a getter/setter pair in the base so it isn't needed.